cypress/integration/login.js

```diff
@@ -7,20 +7,31 @@ describe('Login functionality', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
-        cy.get('[id="loginForm.userId"]').click({force:true})
-        cy.get('[aria-label="Demo User"]').click()
-        cy.get('[id="loginForm.role"]').click({force:true})
-        cy.get('[aria-label="Team Lead"]').click()
-        cy.get('[type="submit"]').click()
+    it('Should be able to login with all roles', function () {
+        const roles = ["User", "Team Lead", "Manager", "Accountant", "Admin"]
+        const tabs = [1, 2, 5, 5, 6]
+        const user = "TestCon User 10"
+        for (let i = 0; i < roles.length; i++) {
```

Comment:
You could try forEach :)

Reply:
@Imlus Correct. But now I have the advantage of looping over the tabs array with the same index.
But this creates a dependency between the roles array and tabs array to be evenly long.